cacheService.js

```diff
@@ -232,17 +239,20 @@ function cacheService(cacheServiceConfig, cacheModules) {
   function checkCacheResponse(key, err, result, type, cacheIndex){
     if(err){
       log(true, 'Error when getting key ' + key + ' from cache of type ' + type + ':', err);
-      if(i < self.caches.length - 1){
-        return {status:'continue'};
-      }
+
+      // thanpolas: this comparison doens't make sense, "i" is not defined at this point.
+      //
+      // if(i < self.caches.length - 1){
```

Comment:
Wow, great catch! This is likely supposed to compare `cacheIndex` rather than `i`.